$NetBSD: patch-src_Makefile_am,v 1.1 2013/03/15 02:01:47 dsainty Exp $

Don't pointlessly chown zms.  Don't setuid zmfix here, let Pkgsrc deal with
that.

--- src/Makefile.am.orig	2010-11-23 01:57:40.000000000 +1300
+++ src/Makefile.am	2010-11-23 01:58:47.000000000 +1300
@@ -124,8 +124,8 @@
 # Yes, you are correct. This is a HACK!
 install-exec-hook:
 	( cd $(DESTDIR)@bindir@; mkdir -p $(DESTDIR)$(cgidir); mv zms $(DESTDIR)$(cgidir) )
-	( cd $(DESTDIR)$(cgidir); chown $(webuser):$(webgroup) zms; ln -f zms nph-zms )
-	( cd $(DESTDIR)@bindir@; chmod u+s zmfix )
+	( cd $(DESTDIR)$(cgidir); ln -f zms nph-zms )
+#	( cd $(DESTDIR)@bindir@; chmod u+s zmfix )
 
 uninstall-hook:
 	( cd $(DESTDIR)$(cgidir); rm -f zms nph-zms )
